### DISCUSSION TITLE
Email Link

### DISCUSSION THREAD

**Discussion by twilson9034** [Budibase/budibase#3422] [Help] [answered]
Has anyone used Javascript for the link to open the native mail client?

I have a link that has the users email just as you would see most email hyperlinks. I would like to be able to complete this action. 

Maybe Budibase can add the ability to make a link an email link that would already be coded to do this.

**Comment by mjashanks** [comment, marked as answer]
Use a `Link` component, with the binding `mailto: {{ Something.emailAddress }}`, and you probably also want to tick "New Tab"...

![image](https://user-images.githubusercontent.com/3524181/142389046-abcf71f5-db9b-44f4-9aab-29e4db46e83e.png)

**Comment by twilson9034** [comment]
I thought I tried that and it didn't work. I'll try again this morning and
update.

**Comment by DevaRyo** [comment]
I had the same question.
With a link it all works fine.

But is it possible with an icon?
I want to have 3 icons at the top of a customer.
Call to phone, call to mobile and mail to email.
It is a bit ugly with Textlinks...
